### ISSUE TITLE
OSError

### ISSUE THREAD

**Issue by bihellzin** [belangeo/pyo#179]
I'm unable to import pyo.
```python
>>> import pyo
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/.local/lib/python3.6/site-packages/pyo/__init__.py", line 25, in <module>
    from . import _linux_wheel_fix_symlinks
  File "/home/user/.local/lib/python3.6/site-packages/pyo/_linux_wheel_fix_symlinks.py", line 88, in <module>
    libasound = ctypes.CDLL(libasound, mode=ctypes.RTLD_GLOBAL)
  File "/usr/lib/python3.6/ctypes/__init__.py", line 348, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /home/user/.pyo/3.6_64/libs/libasound-fb332ab3.so.2.0.0: cannot open shared object file: No such file or directory
```

Tried to use the python3 shell from WSL2 Ubuntu 18.04.

**Comment by zseramnay** [comment]
yes, same here.

`>>> print(sys.executable)
/Library/Frameworks/Python.framework/Versions/3.7/bin/python3
>>> from pyo import *
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pyo/__init__.py", line 28, in <module>
    from .lib import analysis as analysis
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pyo/lib/analysis.py", line 32, in <module>
    from ._core import *
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pyo/lib/_core.py", line 58, in <module>
    from .._pyo import *
ImportError: dlopen(/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pyo/_pyo.cpython-37m-darwin.so, 2): Library not loaded: /usr/local/opt/opus/lib/libopus.0.dylib
  Referenced from: /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pyo/libsndfile.1.dylib
  Reason: image not found`

**Comment by belangeo** [comment]
@bihellzin How did you install pyo? With pip or compiled from source?

@zseramnay Just a note for the future... Not the same issue, this one is fixed with version 1.0.3!

**Comment by belangeo** [comment]
Just installed version 1.0.4 under WSL and libasound is correctly loaded (WSL still need to properly configure to output audio signal).
